configure

```diff
@@ -860,7 +865,7 @@ fi
 fi
 
 # Check for PCLMULQDQ intrinsics
-if test "${ARCH}" = "i686" || test "${ARCH}" = "x86_64"; then
+if test "${ARCH}" = "i386" -o "${ARCH}" = "i686" -o "${ARCH}" = "x86_64"; then
```

Comment:
Hm, seeing this code makes me wonder whether we could clean it up by using something like this instead.
  case $ARCH in i386 | i486 | i586 | i686 | x86_64)
It is kind of abusing the case system, but I think it would be A LOT easier to read than all the clunky if tests.
What do you think?

Reply:
I will amend the patch with the suggested pattern.